tests/Integration/Behaviour/Features/Context/Configuration/PriceDisplayMethodConfigurationFeatureContext.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+namespace Tests\Integration\Behaviour\Features\Context\Configuration;
+
+use Customer;
+use Group;
+
+class PriceDisplayMethodConfigurationFeatureContext extends AbstractConfigurationFeatureContext
+{
+    /**
+     * @Given /^price display method for the group of the customer having email "(.+)" is "(tax included|tax excluded)"$/
+     */
+    public function setPriceDisplayMethodForCustomer(string $customerEmail, string $priceDisplayMethod)
+    {
+        $data = Customer::getCustomersByEmail($customerEmail);
+        $customer = new Customer($data[0]['id_customer']);
```

Comment:
Hi dear @matthieu-rolland,

The user has been fetched. Why the new object?

Reply:
hello @Arman-Hosseini ,

`Customer::getCustomersByEmail` returns the result of a sql query, so $data only contains an array of data, not a customer object